src/createStore.js

```diff
@@ -198,6 +199,49 @@ export default function createStore(reducer, initialState, enhancer) {
     dispatch({ type: ActionTypes.INIT })
   }
 
+  /**
```

Comment:
@blesh @benjamingr  there is https://github.com/mobxjs/mobx/issues/169 but I don't experiment with mobx anymore so I didn't made any progress. Also there is an interop you @benjamingr might find useful https://github.com/chicoxyzzy/rx-mobx